templates/definition/charger/teslalogger.yaml

```diff
@@ -0,0 +1,55 @@
+template: teslalogger
+products:
+  - description:
+      generic: TeslaLogger
+requirements:
+  description:
+    en: Open source Tesla data logger https://github.com/bassmaster187/TeslaLogger, works only with "Teslalogger vehicle"
+    de: Open Source Tesla Datenlogger https://github.com/bassmaster187/TeslaLogger, funktioniert nur in Verbdinung mit "TeslaLogger Fahrzeug"
+params:
+  - name: url
+    required: true
+    example: http://192.0.2.2
+  - name: port
+    example: 5000
+    default: 5000
+  - name: id
+    description:
+      de: TeslaLogger CarID
+      en: TeslaLogger CarID
+    default: 1
+render: |
+  type: custom
+  status:
+    source: combined
+    charging:
+      source: http
+      uri: {{ .url }}:{{ .port }}/currentjson/{{ .id }}
+      jq: .charging
+    plugged:
+      source: go
+      script: |
+        plugged_in && ishome
+      in:
+      - name: plugged_in
+        type: bool
+        config:
+          source: http
+          uri: {{ .url }}:{{ .port }}/currentjson/{{ .id }}
+          jq: .plugged_in
+      - name: ishome
+        type: bool
+        config:
+          source: http
+          uri: {{ .url }}:{{ .port }}/currentjson/{{ .id }}
+          jq: .TLGeofenceIsHome
+  enabled:
+    source: http
+    uri: {{ .url }}:{{ .port }}/currentjson/{{ .id }}
+    jq: .charging
+  enable:
+    source: http
```

Comment:
Ich denke hier brauchts ein switch Plugin- Du musst ja start oder stop aufrufen (falls es die gäbe und abhängig davon wie wir das lösen werden).

Reply:
wir (=TeslaLogger) haben auch getrennte charge_start && charge_stop Endpoints.
beim start_stop muss man true bzw. false senden 
mit charge_start_stop funktioniert heute schon